github/issue_import.go

```diff
@@ -89,7 +89,7 @@ func (s *IssueImportService) Create(ctx context.Context, owner, repo string, iss
 		if ok {
 			decErr := json.Unmarshal(aerr.Raw, i)
 			if decErr != nil {
-				err = decErr
+				return nil, resp, decErr
```

Comment:
I'm concerned about this change in behavior... it looks to me like what we had before was most likely broken (because `i` should not have been returned), but I'm thinking that the original `AcceptedError` should be returned here instead of the attempt to unmarshal the JSON.

Because if we return the `json.Unmarshal` error, the client will have no way of knowing that this is actually an `AcceptedError` meaning that GitHub accepted the call but decided to not yet return a value.

Thoughts?

Reply:
yeah it's unusual here because the err was never used in this can, so we could simply drop the decErr which will be the current behavior

Reply:
Except that if `decErr` is `nil` then we do want to actually return the value of `i`, so we cannot drop it entirely. I'm just saying we should return `aerr` here instead.